presto-spi/src/main/java/com/facebook/presto/spi/function/FunctionHandle.java

```diff
@@ -11,16 +11,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.metadata;
+package com.facebook.presto.spi.function;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import java.util.Objects;
 
-import static com.google.common.base.Preconditions.checkArgument;
 import static java.util.Objects.requireNonNull;
 
+/*
+ * FunctionHandle is a unique handle to identify the function implementation from namespaces.
```

Comment:
Less also add a @Unstable annotation?

Reply:
We cannot use @Unstable or @Beta in spi because they are from other package. But I can add a @apiNote in the JavaDoc here.